README.md

```diff
@@ -4,16 +4,64 @@ Model Assessment and Decision Analysis for Marine Environments web application.
 Prototype Decision Support app for the [ADRIA.jl](https://github.com/open-AIMS/ADRIA.jl)
 platform and CoralBlox model. This project also contains the Reef Guide app.
 
+## Table of Contents
+- [Quick Start](#quick-start)
+- [Angular Developer Setup](#angular-developer-setup)
+- [Development Server](#development-server)
+- [ReefGuide Web API Integration](#reefguide-web-api-integration)
+- [Code Scaffolding](#code-scaffolding)
+- [Configuration](#configuration)
+- [Build](#build)
+- [Deploy](#deploy)
+- [Testing](#testing)
+- [Further Help](#further-help)
+
+## Quick Start
+
+```bash
+# Clone the repository
+git clone https://github.com/open-AIMS/MADAME-app.git
+cd MADAME-app
+
+# For Linux: Install Node with NVM
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
```

Comment:
I don't like copying install instructions like this, this is already out of date (current nvm version is 0.40.2). Also, `nvm` only works on Linux-like environments. nodejs.org has a selector for different OS, I'd just link to https://nodejs.org/en/download

Reply:
I guess the idea is you could copy this whole thing and it would work? But most of these commands should only be run once and don't need to be run at all if you're already done some steps